packages/ai-sandbox/tests/file-diff.test.ts

```
@@ -0,0 +1,112 @@
+import { describe, it, expect } from 'vitest'
```

ESLint

[error] Member 'expect' of the import declaration should be sorted alphabetically.

(sort-imports)